packages/webapp/src/routes/AnimalsRoutes.jsx

```diff
@@ -0,0 +1,35 @@
+/*
+ *  Copyright 2023 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import { Route, Switch, Redirect } from 'react-router-dom';
+import {
+  ANIMALS_INVENTORY_URL,
```

Comment:
Routes file needs the update

```suggestion
  ANIMALS_HOME_URL,
```

Reply:
doh! thanks 🙏 !